test/unit/visitor/sympy_solver.cpp

```diff
@@ -64,6 +64,92 @@ std::vector<std::string> run_sympy_solver_visitor(
     return results;
 }
 
+void compare_blocks(const std::string& result, const std::string& expected) {
+    using namespace pybind11::literals;
+
+    auto locals = pybind11::dict("s1"_a = result, "s2"_a = expected, "is_equal"_a = false);
+    pybind11::exec(R"(
+                    def compare_blocks(s1, s2):
+                        def sanitize(s):
+                            import re
+                            d = {'\[(\d+)\]':'_\\1',  'pow\((\w+), ?(\d+)\)':'\\1**\\2'}
```

Comment:
`a nitpick` : would it be possible to expand some variable names? i.e. to avoid single letter var acmes and expand a bit? e.g. d, d1, d2, s.

Reply:
I am using the names that are already in the nmodl print. We get A[0] -> A_0 for example. I have a feeling that you may be confused: the d that you see here means just digit in regex and the replacement is done out of the usual nmodl framework: this is used only in this function. However, if you think A_0 is too short I will add something, np